src/main/java/com/github/kokorin/jaffree/process/ProcessHandler.java

```diff
@@ -120,6 +121,18 @@ public synchronized ProcessHandler<T> setArguments(final List<String> arguments)
         return this;
     }
 
+    /**
+     * Overrides the default Executor timeout.
+     *
+     * @param executorTimeoutMillis the new Executor timeout in milliseconds
+     * @return this
+     */
+    public synchronized
```

Comment:
`synchronized` can't be used that that. All access paths (both read and write) must come through synchronization block. It's better to declare `executorTimeoutMillis` as volatile since it doesn't require read-and-update semantics. Only independent reads and independent writes.

Reply:
Sorry, no need even for volatile as `ProcessHandler` isn't involved into multithreading.